glean-core/ios/GleanTests/Metrics/JweMetricTests.swift

```diff
@@ -0,0 +1,133 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@testable import Glean
+import XCTest
+
+// swiftlint:disable force_cast
+// REASON: Used in a test
+class JweMetricTests: XCTestCase {
+    // swiftlint:disable line_length
+    private var header: String = "eyJhbGciOiJSU0EtT0FFUCIsImVuYyI6IkEyNTZHQ00ifQ"
```

Comment:
nit: If the test doesn't change these values anywhere, then they should be defined as constant with `let`